src/platform/nrfconnect/wifi/ConnectivityManagerImplWiFi.cpp

```diff
@@ -63,20 +63,6 @@ CHIP_ERROR ConnectivityManagerImplWiFi::_SetWiFiStationMode(ConnectivityManager:
     if (aMode != mStationMode)
```

Comment:
1. This `if` seems unnecessary now.
2. Also, `CHIP_ERROR_INVALID_ARGUMENT` instead of `CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE` should be used above.
3. Note that semantics of this method is different than on other platforms (which actually enable or disable the Wi-Fi interface). It doesn't matter a lot as long as it's not called by the application, but it can be surprising to customers. May be a good idea to file a task to sort it out and align the behavior with other platforms.

Reply:
1. Done.
2. Done.
3. I agree, originally I wanted to stick to the semantics similar to other platforms (although slightly forcefully). Currently it makes not much sense to do it, but yes, we should check if this can lead do some potential problems on customer side and better understand the role of ConnectivityManager entity. JIRA ticket created.